addons/event/views/event_views.xml

```diff
@@ -395,6 +395,26 @@
           <field name="search_view_id" ref="view_registration_search"/>
         </record>
 
+        <!--MASS MAILING-->
+        <act_window
```

Comment:
Every time you use `<act_window>` god kills a kitten. Please think of the kitten.

![](http://digitaldeconstruction.com/wp-content/uploads/2013/01/kitten.jpg)